pages/about.html

```diff
@@ -0,0 +1,126 @@
+                <a href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container-abt">
+            <section class="headline-abt">
+                <h2 class="grtings mb">"Hello Sharing world"</h2>
+                <h2 class="title-abt">
+                    CREATIVE COMMONS<br>GLOBAL SUMMIT 2023
+                </h2>
+                <p class="intro-txt-abt">
+                    The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                    This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                    So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together
+                </p>
+                <p class="intro-txt-abt-1">
+                    Please contact us per Email for nay further questions about CC Global Summit 2023!
+                </p>
+                <a class="email intro-txt-abt-1" href="mailto:sagiecyber@gmail.com">sagiecyber@gmail.com</a>
+            </section>
+        </section>
+    </header>
+    <main>
+        <section class="main-abt">
+            <h2 class="title-abt-1">
+                Creative Commons Global Summit 2023 Logo
+            </h2>
+            <p class="intro-txt-abt-2">
+                The logo of CC Global Summit 2023 was decided through the logo competition from 8, June to 7, July. 
+                After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize. 
+                After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+            </p>
+            <a href="#" class="logo-abk">BLK-NG</a>
+        </section>
+        <section class="main-abt-2">
+            <h2 class="title-abt-1">See the past Black Ng Global Summits</h2>
+            <p class="intro-txt-abt-2">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            <section class="pass-celebrate">
+               <article class="p-c">
+                    <div>
+                        <h2>2015</h2>
+                        <p class="intro-txt-abt-2">BLK-NG Summit in Lagos</p>
+                    </div>
+               </article> 
+               <article class="p-c">
+                    <div>
+                        <h2>2020</h2>
+                        <p class="intro-txt-abt-2">BLK-NG Summit in Edo</p>
+                    </div>
+                </article> 
```

Comment:
- [ ] 👏 Great job on implementing the images backgroud for both mobile and desktop versions! However, to match the design requirements, the image size should be increased. Please update the width and height of the images to align with the design.

By making this adjustment, your project will look even more visually appealing and in line with the intended design. Keep up the good work, and happy coding! 🌟💻🚀